jvb/src/main/java/org/jitsi/videobridge/Endpoint.java

```diff
@@ -289,8 +290,15 @@ public void trace(@NotNull Function0<Unit> f)
                 }
             });
 
-        BitrateController.EventHandler bcEventHandler = new BitrateController.EventHandler()
+        org.jitsi.videobridge.cc.allocation.EventHandler bcEventHandler
+                = new org.jitsi.videobridge.cc.allocation.EventHandler()
```

Comment:
I think making the event handler a static inner class of the class firing the events is what we've done elsewhere and makes the scope of it a bit more obvious.